run-a-cluster/1-DOC.yaml

```diff
@@ -95,10 +106,13 @@ services:
       - SYSTEM_MANAGER_PORT=10000
       - CLOUD_MONGO_URL=localhost
       - CLOUD_MONGO_PORT=10007
+      - RESOURCE_ABSTRACTOR_URL=localhost
+      - RESOURCE_ABSTRACTOR_PORT=11011
       - REDIS_ADDR=redis://:cloudRedis@localhost:6379
     depends_on:
       - redis
       - mongo_root
```

Comment:
cloud_scheduler no longer depends on mongo_root.

Reply:
we could also remove the mongo env from cloud_scheduler

Reply:
ty for noticing, removed those ;)